doc/api/deprecations.md

```diff
@@ -206,11 +206,12 @@ See [`Intl.Segmenter`](https://github.com/tc39/proposal-intl-segmenter).
 <a id="DEP0018"></a>
 ### DEP0018: Unhandled promise rejections
 
-Type: Runtime
+Type: End-of-Life
 
-Unhandled promise rejections are deprecated. In the future, promise rejections
-that are not handled will terminate the Node.js process with a non-zero exit
-code.
+Any rejected promise that is unhandled and garbage collected is going to exit
+the process similar to uncaught exceptions. Please make sure to always handle
```

Comment:
This is not true if the default setting is `WARN`

Reply:
See https://github.com/nodejs/node/pull/20097#issuecomment-396326405